sources/europe/fr/PCRS.geojson

```diff
@@ -0,0 +1,2289 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "fr.pcrs",
+        "name": "PCRS raster",
+        "type": "tms",
+        "url": "https://wxs.ign.fr/ortho/geoportail/wmts?SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER=PCRS.LAMB93&STYLE=normal&FORMAT=image/jpeg&tileMatrixSet=LAMB93_5cm_EPSG&tileMatrix={zoom}&tileRow={y}&tileCol={x}",
+        "best": true,
+        "country_code": "FR",
+        "attribution": {
+            "url": null,
+            "text": "PCRS - République Française",
+            "required": true
+        },
+        "icon": "https://pcrs.beta.gouv.fr/images/icons/map.png",
+        "max_zoom": 21,
+        "min_zoom": 3,
+        "available_projections": [
+            "EPSG:2154"
+        ],
+        "license_url": "https://www.etalab.gouv.fr/licence-ouverte-open-licence/",
```

Comment:
The `license_url` should point to a website that specifies the license of the data, not the license itself. E.g. ideally on the site linked here, it should say dataset PCRS rasters has license X.

Reply:
Indeed, I only know https://geoservices.ign.fr/pcrs which isn't clear regarding the license.

Be sure I'll ask to complete it with appropriate mention next week.
All imagery from IGN/public services is Etalab/Open Licence v2 actually.